digits/pretrained_model/tasks/upload_pretrained.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+from __future__ import absolute_import
+
+import base64
+from collections import OrderedDict
+import h5py
```

Comment:
For future PRs, put this import in a second group because it doesn't come with a standard Python installation.
https://www.python.org/dev/peps/pep-0008/#imports

Don't bother to fix it now, though.